src/main/java/com/microsoft/aad/msal4j/DefaultHttpClient.java

```diff
@@ -46,6 +50,8 @@ private HttpResponse executeHttpPost(HttpRequest httpRequest) throws Exception {
         configureAdditionalHeaders(conn, httpRequest);
         conn.setRequestMethod("POST");
         conn.setDoOutput(true);
+        conn.setConnectTimeout(defaultConnectTimeout);
+        conn.setReadTimeout(defaultReadTimeout);
```

Comment:
might be better to set timeouts in openConnection function

Reply:
It's now in the openConnection() function, and the two constants are all upper-case.